packages/ui/components/overlays/src/OverlayController.js

```diff
@@ -121,6 +121,8 @@ export class OverlayController extends EventTarget {
     /** @private */
     this.__sharedConfig = config;
 
+    this.blockEscKeyHandler = false;
```

Comment:
If we want to make it part of OverlayController config:
- it should be added to OverlayConfig type (under User interaction): https://github.com/ing-bank/lion/blob/master/packages/ui/components/overlays/types/OverlayConfig.ts#L51
- in the past, we introduced a specific naming convention (3rd person instead of usual imperative). Would be nice to stay consistent => `blocksEscKeyHandler`
- to align with rest of config options
  - should be part of `_defaultConfig`
  - should have getter  reading from config (and no setter)

Reply:
Also, having a closer look, shouldn't this be the default behavour of `hidesOnEsc`?

Only `hidesOnOutsideEsc` should close all ovelays

Reply:
The `blocksEscKeyHandler` is not a configurable option, it is set internally. So i don't think it should be part of the config options.

Reply:
Ah nice, you made it a protected variable now